src/signingClient.spec.ts

```diff
@@ -4,9 +4,10 @@ import { DirectSecp256k1HdWallet, AccountData } from '@cosmjs/proto-signing';
 import { TxRaw } from 'cosmjs-types/cosmos/tx/v1beta1/tx';
 import { ISCNSigningClient } from './signingClient';
 import { computeTransactionHash } from './tests/utils';
-import { mnemonic } from './tests/key.json';
+import { mnemonic, address1 } from './tests/key.json';
```

Comment:
Seems `mnemonic` and `address` are a pair, and `mnemonic1` and `address1` are another pair. Will it be easier to realize the fact if we use the names `mnemonic1`, `address1`, `mnemonic2`, and `address2`?

Reply:
`mnemonic0` then